src/SOS/SOS.UnitTests/ConfigFiles/Unix/Debugger.Tests.Config.txt

```diff
@@ -117,6 +117,39 @@
         </Option>
       </Options>
     </Option>
+    <!--
+        DotnetDumpCommands (runs on every runtime except netcore2.1 on Linux)
+      -->
+    <Option>
+      <Options>
+        <Option>
+          <TestName>DotnetDumpCommands</TestName>
+        </Option>
+      </Options>
+      <Options>
+        <Option Condition="'$(RuntimeVersionLatest)' != ''">
+          <BuildProjectFramework>$(BuildProjectFrameworkLatest)</BuildProjectFramework>
+          <RuntimeFrameworkVersion>$(RuntimeVersionLatest)</RuntimeFrameworkVersion>
+          <SetHostRuntime>$(DotNetRoot)/shared/Microsoft.NETCore.App/$(RuntimeFrameworkVersion)</SetHostRuntime>
+        </Option>
+        <Option Condition="'$(RuntimeVersion50)' != ''">
+          <BuildProjectFramework>net5.0</BuildProjectFramework>
+          <RuntimeFrameworkVersion>$(RuntimeVersion50)</RuntimeFrameworkVersion>
+        </Option>
+        <Option Condition="'$(RuntimeVersion31)' != ''">
+          <BuildProjectFramework>netcoreapp3.1</BuildProjectFramework>
+          <RuntimeFrameworkVersion>$(RuntimeVersion31)</RuntimeFrameworkVersion>
+          <SetHostRuntime>$(DotNetRoot)/shared/Microsoft.NETCore.App/$(RuntimeFrameworkVersion)</SetHostRuntime>
+        </Option>
+      </Options>
+      <Options>
+        <Option Condition="'$(OS)' != 'Linux'">
```

Comment:
The "'$(RuntimeVersion21)' != ''" condition still needs to be checked here, too.  There are scenarios where RuntimeVersion21 is blank.  You need to do something like this:

First create a variable at the top of the file:
```
  <TestDumpCommands>true</TestDumpCommands>
  <TestDumpCommands Condition="'$(RuntimeVersion21)' == ''">false</TestDumpCommands>
  <TestDumpCommands Condition="'$(OS)' == ''Linux">false</TestDumpCommands>
```

And use `TestDumpCommands` as the condition here.

Reply:
Sorry, I forgot this check... I updated with the new variable as well (nice way to do some logic operations)